src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.md

```diff
@@ -73,5 +73,6 @@ Configured to show a table:
     sqlOperator='='
     sql='SELECT name, assay, study, dataType, tissue, consortium FROM syn11346063.14'
     downloadCartPageUrl='#/Other%20Components?id=downloadcartpage'
+    hideSqlEditorControl={false}
```

Comment:
hideSqlEditorControl defaults to true for now, so you would need to explicitly set hideSqlEditorControl to false in order to see the new icon and access the SqlEditor.